src/Search.js

```diff
@@ -19,16 +28,85 @@ class Search extends Component {
 
     render() {
         return (
-            <div>
-                <h1>Search</h1>
-                <p>Search page body content</p>
-                <form onSubmit={this.handleSubmit}>
-                    <label>
-                    <input type="text" placeholder="Search keyword" value={this.state.value} onChange={this.handleChange} />
-                    </label>
-                    <input type="submit" value="Submit" />
-                </form>
-            </div>
+            <Container fluid>
```

Comment:
Looks great with Bootstrap! It would definitely be good to start adding comments to highlight what parts of the code correspond to the major UI components!